tools/test-setup.sh

```diff
@@ -142,7 +142,13 @@ if [[ "${OS:-}" == "darwin" && "${SKIP_PODMAN:-}" != '1' ]]; then
     podman machine ls --format '{{.Name}} {{.Running}}' --noheading | grep podman-machine-default | grep true || {
         # do not use full path as it varies based on architecture
         # https://github.com/containers/podman/issues/10824#issuecomment-1162392833
-        "qemu-system-${MACHTYPE}" -machine q35,accel=hvf:tcg -cpu host -display none INVALID_OPTION || true
+        # MACHTYPE can look like x86_64 or x86_64-apple-darwin20.6.0
+        if [[ $MACHTYPE == "x86_64*" ]] ; then
+            ARCH=x86_64
+        else
+            ARCH=aarch64
```

Comment:
Is it `aarch64` (double `a` at start) the right value?

Reply:
Yep.